HopsanGUI/GUIObjects/AnimatedComponent.cpp

```diff
@@ -106,6 +107,17 @@ AnimatedComponent::AnimatedComponent(ModelObject* unanimatedComponent, Animation
                             tempPortName = mpModelObject->getPort(portName)->getConnectedPorts().first()->getName();
                         }
                         mpData->last().append(mpAnimationWidget->getPlotDataPtr()->copyVariableDataVector(makeFullVariableName(mpModelObject->getParentSystemNameHieararchy(), tempComponentName, tempPortName, dataName),generation));
+
+                        const auto& dataValues = mpData->last().last();
```

Comment:
In this case there should be an & after auto, to ensure that dataValue is not a copy, we want to use a const reference to the actual value.
This would be equivalent to the [i] call that was used before

Reply:
Hmm my app seem to add the comments to the wrong line.
I mean the one in the for loop